File: Ejercicio3Javascript/Fase4.js
var ciudades = ['Barcelona', 'Valencia', 'Malaga', 'Cadiz'];

var ciudad1 = new Array (ciudades[0].length);
var ciudad2 = new Array (ciudades[1].length);
var ciudad3 = new Array (ciudades[2].length);
var ciudad4 = new Array (ciudades[3].length);

for (let i = ciudades[0].length - 1; i >= 0; i--) {
  ciudad1[i] = ciudades[0].charAt (i);
  console.log (ciudad1[i]);
}

for (let i = ciudades[1].length - 1; i >= 0; i--) {
  ciudad2[i] = ciudades[1].charAt (i);
  console.log (ciudad2[i]);
}

for (let i = ciudades[2].length - 1; i >= 0; i--) {
  ciudad3[i] = ciudades[2].charAt (i);
  console.log (ciudad3[i]);
}

for (let i = ciudades[3].length - 1; i >= 0; i--) {
  ciudad4[i] = ciudades[3].charAt (i);
  console.log (ciudad4[i]);
}

File: Ejercicio3NomCiutatsJava/src/fases/Fase4.java
package fases;

public class Fase4 {

	public static void main(String[] args) {
		
		
		String [] cities = {"Barcelona", "Madrid", "Valencia", "Málaga", "Cadiz", "Santander"};
	
		
		char [] city1 = new char[cities[0].length()];
		char [] city2 = new char[cities[1].length()];
		char [] city3 = new char[cities[2].length()];
		char [] city4 = new char[cities[3].length()];
		char [] city5 = new char[cities[4].length()];
		char [] city6 = new char[cities[5].length()];
		 
				
		for(int i=cities[0].length()-1;i>=0; i--) {
		 city1[i]= cities[0].charAt(i);
		 System.out.print(city1[i]);
			}
		
		System.out.println("");
		
		for(int i=cities[1].length()-1;i>=0; i--) {
			 city2[i]= cities[1].charAt(i);
			 System.out.print(city2[i]);
				}
		
		System.out.println("");
		
		for(int i=cities[2].length()-1;i>=0; i--) {
			 city3[i]= cities[2].charAt(i);
			 System.out.print(city3[i]);
				}
		
		System.out.println("");
		
		for(int i=cities[3].length()-1;i>=0; i--) {
			 city4[i]= cities[3].charAt(i);
			 System.out.print(city4[i]);
				}
		
		System.out.println("");
		
		for(int i=cities[4].length()-1;i>=0; i--) {
			 city5[i]= cities[4].charAt(i);
			 System.out.print(city5[i]);
				}
		
		System.out.println("");
		
		for(int i=cities[5].length()-1;i>=0; i--) {
			 city6[i]= cities[5].charAt(i);
			 System.out.print(city6[i]);
				}
		
		System.out.println("");

		}
	}


